README.md

```diff
@@ -59,67 +27,83 @@ sudo apt-get update && sudo apt-get install ros-ROS_DISTRO-microstrain-inertial-
 For more information on the ROS distros and platforms we support, please see [index.ros.org](https://index.ros.org/r/microstrain_inertial/github-LORD-MicroStrain-microstrain_inertial/#noetic)
```

Comment:
not a change but noticed this specifically links to noetic - probably doesn't matter, looks like if you leave the param off entirely it just defaults to the humble tab.

Reply:
Yes, that is on purpose since humble is ROS2.